README.md

```diff
@@ -1,25 +1,17 @@
 # devbase
 
-This repo contains files needed for a Outreach repository. This is currently used as a library by bootstrap for shell scripts and other configuration.
+[![go.dev reference](https://img.shields.io/badge/go.dev-reference-007d9c?logo=go&logoColor=white)](https://pkg.go.dev/github.com/getoutreach/devbase)
+[![Generated via Bootstrap](https://img.shields.io/badge/Outreach-Bootstrap-%235951ff)](https://github.com/getoutreach/bootstrap)
+[![Coverage Status](https://coveralls.io/repos/github/getoutreach/devbase/badge.svg?branch=main)](https://coveralls.io/github//getoutreach/devbase?branch=main)
 
-The goal of this is to have one place for all scripts / etc needed to run outreach services and that this gets pulled in as needed.
+A collection of scripts and ci configuration
 
-## How to use a custom build of `devbase`
+## Contributing
 
-### Bootstrap 
+Please read the [CONTRIBUTING.md](CONTRIBUTING.md) document for guidelines on developing and contributing changes.
 
-To test a custom build of `devbase` with bootstrap, simply modify `bootstrap.lock` to point to your branch / version. 
+## High-level Overview
 
-**Example**: To use `jaredallard/feat/my-cool-feature` instead of `v1.8.0`, you'd update `versions.devbase` to the former. Full example below:
```

Comment:
@jaredallard did you mean to remove this from README?

Reply:
Good catch, adding back